client/src/pages/SearchResultsUser.js

```diff
@@ -1,23 +1,53 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import { Typography } from "@material-ui/core";
-import SearchList from "../Components/SearchResList";
+import SearchResList from "../Components/SearchResList";
 import { useLocation } from "react-router-dom";
 
 function SearchResultsUser() {
 
-// Will get the query from the URL: https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed
-  const useQuery = () => {
+const [results, setResults] = useState(null);  
+const [error, setError] = useState(null);
+const useQuery = () => {
     return new URLSearchParams(useLocation().search);
   }
 
-  const query = useQuery().get("query");
```

Comment:
Please put this line back, it handles getting the query from that the search bar that the user entered. We'll need it to decide what we are calling.